app/build.gradle.kts

```diff
@@ -35,6 +35,17 @@ android {
         }
     }
 
+    flavorDimensions.add("releaseType")
```

Comment:
can't you use: flavorDimensions("releaseType")?

Reply:
And als value maybe: type

Reply:
it s deprecated